client/src/browser-table.html

```diff
@@ -268,10 +265,7 @@ <h1 label>Browser support</h1>
         <a title="Stable" href="https://www.chromestatus.com/feature/4667415417847808"></a>
         <a title="Stable" href="https://www.chromestatus.com/feature/4667415417847808"></a>
         <a title="Stable" href="https://webkit.org/status/#feature-shadow-dom"></a>
-        <a href="https://platform-status.mozilla.org/#shadow-dom">
-          <div title="Polyfill"></div>
-          <div title="Developing"></div>
-        </a>
+        <a title="Stable" href="https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/63"></a>
```

Comment:
Same here: https://platform-status.mozilla.org/#shadow-dom
```suggestion
        <a title="Stable" href="https://platform-status.mozilla.org/#shadow-dom"></a>
```